ansible/envs/prod/group_vars/playground.yml

```diff
@@ -6,7 +6,7 @@ ssm_playground: "{{ lookup('aws_ssm', '/prod/ansible/playground/', region='us-we
 vars_extra_sudo_users:
   - shep
 
-vars_playground_domain: play-1.infra.rust-lang.org
```

Comment:
This is going away and will be a conflict. Will want to rebase on my upcoming ansible changes.

Reply:
I reverted this change